style.css

```diff
@@ -0,0 +1,28 @@
+
+.container{
+    margin: 0 auto;
+    max-width: 1325px;    
+}
+
+
+header {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 15px;
+}
+
+
+.input-search{  
+     border-radius: 20px;
+     font-family: Muli-Regular;
+     font-size: 21px;
+     background: url(images/shape.png);
+     background-repeat: no-repeat;
+     background-position: center left 10px;
+     padding-left: 40px;
+     height: 40px; 
```

Comment:
It's missing the border, now is showing an incorrect one and is not matching with the design, this is the border that you need to use: `border: 3px solid #f8f8f8;`